### DISCUSSION TITLE
AVB frame routing on the switch

### DISCUSSION THREAD

**Discussion by wkd88347146** [CoRE-RG/CoRE4INET#41] [Q&A] [answered]
Hi.
I want to transmit AVB frame in Ring topology.
I know how Q frames can be routed through the address table on the switch. However, I don't know how to specify the AVB frame path on the switch, so I want to know.
I need your help. thank you :)

**Comment by Musteblume** [comment, marked as answer]
Hi,
you can define static entries for the SRTable as well. In examples/avb_AS6802/static_network is an example which uses static Stream entries.
Hope this helps.
Best regards

**Reply by wkd88347146** [reply]
Thank you for your answer. I solved that problem using SRPtable. :)
